### PR TITLE
Remove red padding around output pane.

### PR DESCRIPTION
**Pull request by juliuscrn**
Removes red padding of output pane introduced by #1995 .

### CODE REVIEW

**Review by captainsafia — APPROVED** [review]

File: styles/hydrogen.less [hunk 1]
```diff
@@ -289,20 +289,19 @@ svg:first-child {
     font-family: monospace;
     &:not(.scroll-list) {
       flex: 1;
-      border: solid 2px red;
```
Comment (BenRussert) [inline styles/hydrogen.less:292]: I have a bad habit of using this almost like `console.log` debugging but for css. Sorry about that!

**Comment by BenRussert** [comment]
Congrats on your first PR @juliuscrn, I hope you stick around!

File: styles/hydrogen.less (hunk 1 above)
Comment (juliuscrn) [reply styles/hydrogen.less:292]: That bad habit kick started my open source development career! ;-)

**Comment by juliuscrn** [comment]
@BenRussert thank you, I hope to contribute more!

**Pull request merged**